config/plugins/interactive_environments/rstudio/config/rstudio.ini.sample

```diff
@@ -0,0 +1,11 @@
+[main]
+# This cannot be changed
+password_auth = True
+# Other
+apache_urls = False
+ssl = False
+
+[docker]
+command = docker
+image = erasche/docker-rstudio-notebook:dev
+
```

Comment:
My intention was to mirror @bgruening's. Whoops. Apparently I didn't turn on the `:dev` tag. Image should be building now. **Thank you** for catching that.